extensions/resource-deployment/src/services/tools/azdataTool.ts

```diff
@@ -29,7 +29,7 @@ export class AzdataTool extends ToolBase {
 	}
 
 	get description(): string {
-		return localize('resourceDeployment.AzdataDescription', "Bootstraps and manages the Big Data Cluster");
+		return localize('resourceDeployment.AzdataDescription', "Azure Data command line interface");
```

Comment:
Did you get this from Travis/Alan? The linked issue mentions having the description be `Deploy, manage and use Azure Arc enabled data services and Big Data Clusters`